android/src/main/java/dev/robingenz/capacitorjs/plugins/firebase/auth/handlers/AppleAuthProviderHandler.java

```diff
@@ -0,0 +1,126 @@
+package dev.robingenz.capacitorjs.plugins.firebase.auth.handlers;
+
+import android.util.Log;
+import com.getcapacitor.JSArray;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.PluginCall;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthCredential;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.OAuthCredential;
+import com.google.firebase.auth.OAuthProvider;
+import dev.robingenz.capacitorjs.plugins.firebase.auth.FirebaseAuthentication;
+import dev.robingenz.capacitorjs.plugins.firebase.auth.FirebaseAuthenticationPlugin;
+import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
+import java.nio.charset.CharsetDecoder;
+import java.nio.charset.CodingErrorAction;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.List;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class AppleAuthProviderHandler {
+
+    private final FirebaseAuthentication pluginImplementation;
+    private String nonce;
```

Comment:
```suggestion
    private String currentNonce;
```

In the iOS implementation this variable is currently called `currentNonce` (see [here](https://github.com/robingenz/capacitor-firebase-authentication/blob/743935511a650a7945150a332499468bf2121c24/ios/Plugin/Handlers/AppleAuthProviderHandler.swift#L10)). It would be good if both implementations are as consistent as possible.